api/voice-react-native-sdk.api.md

```diff
@@ -1046,6 +1172,16 @@ export namespace Voice {
         export type Registered = () => void;
         export type Unregistered = () => void;
     }
+    export type RunPreflightTestOptions = Partial<{
+        [Constants.CallOptionsKeyIceServers]: CallOptions.IceServer[];
+        [Constants.CallOptionsKeyIceTransportPolicy]: CallOptions.IceTransportPolicy;
+        [Constants.CallOptionsKeyPreferredAudioCodecs]: CallOptions.AudioCodec;

```

Comment:
@ramapal @dipankadas are these the only option we provide on mobile for the preflight api?

Reply:
ok